order/client.go

```diff
@@ -285,6 +286,51 @@ func (i *Iter) Order() *stripe.Order {
 	return i.Current().(*stripe.Order)
 }
 
+// Return returns all or part of an order.
+// For more details see https://stripe.com/docs/api#return_order.
+func Return(id string, params *stripe.OrderReturnParams) (*stripe.OrderReturn, error) {
+	return getC().Return(id, params)
+}
+
+// Return returns all or part of an order.
+// For more details see https://stripe.com/docs/api#return_order.
+func (c Client) Return(id string, params *stripe.OrderReturnParams) (*stripe.OrderReturn, error) {
+	var body *url.Values
+	var commonParams *stripe.Params
+
+	if params != nil {
+		body = &url.Values{}
+
+		if len(params.Items) > 0 {
+			for _, item := range params.Items {
```

Comment:
I also just wanted to point out that it seems like any multi-item form arrays may be generally broken in the Go bindings right now (including here). See #251 for more details. I'll try to get this one addressed soon.